securedrop_export/entrypoint.py

```diff
@@ -1,18 +1,54 @@
+import logging
 import os
 import shutil
 import sys
 
+from logging.handlers import TimedRotatingFileHandler
+from securedrop_export import __version__
 from securedrop_export import export
 from securedrop_export import main
 
 CONFIG_PATH = "/etc/sd-export-config.json"
+DEFAULT_HOME = os.path.join(os.path.expanduser("~"), ".securedrop_export")
 
+def configure_logging():
+    """
+    All logging related settings are set up by this function.
+    """
+    log_folder = os.path.join(DEFAULT_HOME, 'logs')
+    if not os.path.exists(log_folder):
+        os.makedirs(log_folder)
+
+    log_file = os.path.join(DEFAULT_HOME, 'logs', 'export.log')
+
+    # set logging format
+    log_fmt = ('%(asctime)s - %(name)s:%(lineno)d(%(funcName)s) '
+               '%(levelname)s: %(message)s')
+    formatter = logging.Formatter(log_fmt)
+
+    handler = TimedRotatingFileHandler(log_file)
+    handler.setFormatter(formatter)
+    handler.setLevel(logging.DEBUG)
+
+    # set up primary log
+    log = logging.getLogger()
+    log.setLevel(logging.DEBUG)
+    log.addHandler(handler)
 
 def start():
+    try:
+        configure_logging()
+    except Exception:
+        msg = "ERROR_LOGGING"
+        my_sub.exit_gracefully(msg)
```

Comment:
Careful: `my_sub` isn't defined at this point, so the `exit_gracefully` call will raise an exception of its own. `flake8` catches this mistake, but we don't run flake8 in CI for this repo—perhaps we should add it. Would be simple enough to copy/paste from e.g. https://github.com/freedomofpress/securedrop-workstation/blob/bbebfc0f0f1fd45bab7956e3d9d4412c434532ca/Makefile#L127-L133